man/fido_cbor_info_new.3

```diff
@@ -137,6 +139,12 @@ function returns the maximum message size attribute of
 .Fa ci .
 .Pp
 The
+.Fn fido_cbor_info_maxcredbloblen
+function returns the maximum credBlob length in bytes supported by the
```

Comment:
`.Dq credBlob` for consistency with the way we refer to `largeBlob`. feel free to be more specific if you feel like, e.g. `FIDO 2.1 .Dq credBlob`

Reply:
fixed in 3e7e37e1ead79f0539b18bafee58039cc618c3ea